Cineaste/Extension/Date+Formatter.swift

```diff
@@ -9,15 +9,24 @@
 import UIKit
 
 extension Date {
-    var formatted: String {
+    private static let dateFormatter = { () -> DateFormatter in
         let formatter = DateFormatter()
-        formatter.dateFormat = "dd.MM.YYYY"
-        return formatter.string(from: self)
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        return formatter
+    }()
+
+    var formatted: String {
+        Date.dateFormatter.dateFormat = "dd.MM.YYYY"
```

Comment:
We could also just use something like `dateFormatter.dateStyle = .short` to make the date formatting automatically adapt to the given region.

Reply:
That's true. I updated the date format so it is displayed correctly to the given locale.